src/main/java/nextstep/qna/domain/Question.java

```diff
@@ -65,28 +66,43 @@ public NsUser getWriter() {
 
     public void addAnswer(Answer answer) {
         answer.toQuestion(this);
-        answers.add(answer);
+
+        List<Answer> originAnswers = this.answers.getValue();
+        originAnswers.add(answer);
+
+        answers = new Answers(originAnswers);
```

Comment:
`Answers`에서 값을 꺼내서 처리하는 대신에, 메시지를 보내서 처리해보면 어떨까요?
```suggestion
        answers.add(answer);
```